orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls

```diff
@@ -24,10 +24,9 @@
       private_address: {{ host.private_address }}
 
 include_access_config:
-  file.replace:
-    - name: /etc/unbound/unbound.conf
-    - pattern: '#include: "otherfile.conf"'
-    - repl: 'include: "/etc/unbound/access.conf"'
+# file.replace tends to take 5 seconds when run for the first time due to tons of modules being loaded in salt 3000.2. Hence a sed.
```

Comment:
I think if it tends to take a one time toll on us it shouldn't be an issue as we have a bunch of `file.replace`. So we should eliminate all of them or leave this one here also. I'm voting for leaving this one as it's more readable and error proof

Reply:
sure ...

Reply:
Removed. The deletion of vsphere brings such salt operations from 5s to ~2s. This would take it further down to a few milliseconds. Can be addressed later.